prow/cmd/generic-autobumper/bumper/bumper.go

```diff
@@ -473,7 +473,8 @@ func MakeGitCommit(remote, remoteBranch, name, email string, stdout, stderr io.W
 }
 
 func makeGerritCommit(summary, commitTag, changeId string) string {
-	return fmt.Sprintf("%s\n\n[%s]\n\nChange-Id: %s", summary, commitTag, changeId)
+	//Gerrit commits do not recognize "&#x2011;" as NON-BREAKING HYPHEN, so just replace with a regular hyphen.
```

Comment:
What is creating the non-breaking hyphens? Can we make this tool stop doing that?

Reply:
https://github.com/kubernetes/test-infra/blob/05d1f8276331b067480d524e3343edfe0f410bde/prow/cmd/generic-autobumper/helper.go#L46 date formating that we were using for github commits to prevent line-wrapping. We could also just remove this if we do not care about line wrapping in the github commits.